src/Components/Nav/CommunityNav/CommunityNav.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import "./CommunityNav.scss";
+
+class CommunityNav extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      display: "flex",
+      nonMemberDisplay: "flex",
+      memberDisplay: "none",
+    };
+  }
+
+  placeholderFocusEvent = () => {
+    this.setState({
+      display: "none",
+    });
+  };
+
+  placeholderBlurEvent = (e) => {
+    if (!e.target.value) {
+      this.setState({
+        display: "flex",
+      });
+    }
+  };
+
+  handleSignInEvent = () => {
+    this.setState({
+      nonMemberDisplay: "none",
+      memberDisplay: "flex",
+    });
+  };
+
+  handleSignOutEvent = () => {
+    this.setState({
+      nonMemberDisplay: "flex",
+      memberDisplay: "none",
+    });
+  };
+
+  render() {
+    return (
+      <nav className="communityNav">
+        <section className="searchSection">
+          <div className="navMenuWrapper">
+            <img className="logo" alt="Logo" src="/images/styleWeLogo.png" />
+            <div className="menuWrapper">
+              <span>#OOTD</span>
+              {/* 커뮤니티 메인 페이지 완성 후, <Link>태그 사용 예정 */}
+              <span>STORE</span>
+              {/* 스토어 상세 페이지 완성 후, <Link>태그 사용 예정 */}
+            </div>
+          </div>
+          <div className="navSearchBarWrapper">
+            <div className="searchBar">
+              <input
+                className="searchInput"
+                type="search"
+                onFocus={this.placeholderFocusEvent}
+                onBlur={this.placeholderBlurEvent}
+              />
+              <div
+                className="searchBarPlaceHolder"
+                style={{ display: this.state.display }}
+              >
+                <img
+                  className="searchIcon"
+                  alt="search icon"
+                  src="/images/searchIcon.svg"
+                />
+                <span>스타일과 상품을 검색해 보세요</span>
+              </div>
```

Comment:
여기서 조건부 렌더링으로 구현하면 됩니다.
다른 두 부분도 조건부 렌더링으로 구현해주세요!

```suggestion
              { display &&
                (
                  <div
                    className="searchBarPlaceHolder"    
                  >
                    <img
                      className="searchIcon"
                      alt="search icon"
                      src="/images/searchIcon.svg"
                    />
                    <span>스타일과 상품을 검색해 보세요</span>
                  </div>
                )
              }
```

Reply:
네 알겠습니다!...

Reply:
```suggestion
                  <input
                    className="searchInput"
                    type="search"
                    onFocus={this.placeholderFocusEvent}
                    onBlur={this.placeholderBlurEvent}
                  />
                  <div
                    className="searchBarPlaceHolder"
                    style={{ display: this.state.display }}
                  >
                    <img
                      className="searchIcon"
                      alt="search icon"
                      src="/images/searchIcon.svg"
                    />
```
해당 부분 질문이 있습니다!
제가 구현한 사항은, input과 className="searchBarPlaceHolder"의 <div>태그가 형제사이가 아니며,

<input>에서 onFocus 시, className="searchBarPlaceHolder" 의 (display: flex) 되도록 하고,
onBlur && input값이 없을 때, className="searchBarPlaceHolder" 의 (display: none) 으로 변경 되도록 설정하였습니다.

오늘 조건부 렌더링 수정하고자 하는데 도통 답을 찾을 수 가 없어서....힌트를 받을 수 있을까요?